src/common.rs

```diff
@@ -85,7 +90,9 @@ pub enum Type<'a, T: Text<'a>> {
     NonNullType(Box<Type<'a, T>>),
 }
 
+
 impl Number {
+    #[cfg(not(feature = "as_i32"))]
```

Comment:
Feature flags should be additive. We can still provide `as_i64` even with an `i32` because that has an `impl Into`.